azure-pipelines-server.yml

```diff
@@ -0,0 +1,22 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '16.x'
+  displayName: 'Install Node.js'
+
+- script: 
+    cd fujiji-server/
+    npm install docker compose up
```

Comment:
Not sure why we need to run the application in the CI pipeline.

So, I think 
```
npm install
```

 is enough

Reply:
because this pipeline creates build artifacts which is further used for deployment.